common/src/test/java/com/tc/net/core/TCWorkerCommManagerTest.java

```diff
@@ -67,6 +67,7 @@
 import java.util.Properties;
 import java.util.concurrent.Callable;
 import org.junit.Ignore;
+import org.terracotta.utilities.test.net.PortManager;
 
 public class TCWorkerCommManagerTest extends TCTestCase {
```

Comment:
The majority of changes in here appear to be formatting related? Would it be better to stick to the minimum necessary changes here?

Reply:
As I noted in the comments for the review, many of the files had trailing whitespace which my editor dutifly removed when the file was saved.  I'm not inclined to put it back.